keras_cv/layers/preprocessing3d/group_points_by_bounding_boxes.py

```diff
@@ -21,6 +21,7 @@
 BOUNDING_BOXES = base_augmentation_layer_3d.BOUNDING_BOXES
 OBJECT_POINT_CLOUDS = base_augmentation_layer_3d.OBJECT_POINT_CLOUDS
 OBJECT_BOUNDING_BOXES = base_augmentation_layer_3d.OBJECT_BOUNDING_BOXES
+BOX_LABEL_INDEX = base_augmentation_layer_3d.BOX_LABEL_INDEX
```

Comment:
For these constant assignments -- would it make more sense to just import these symbols directly instead?

e.g. from keras_cv.layers.preprocessing3d.base_augmentation_layer_3d import BOX_LABEL_INDEX

Your call -- I don't have a strong opinion on this.

Reply:
Let's keep it this way for now and decide once all the augmentations are released?

Reply:
sgtm